ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -1003,6 +1003,39 @@ def test_paddle_heaviside(
     )
 
 
+# inner
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.inner",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
+        num_arrays=2,
+        min_num_dims=1,
+        max_num_dims=1,
```

Comment:
According to the documentation `inner` should work for N-D tensors, so we need to test all cases.
Other than that, looks good!

Reply:
Thanks! Should be fixed now hopefully :)

Reply:
The CI checks show all the backends are failing. I would suggest running the test locally to debug it.